packages/cactus-plugin-ledger-connector-iroha/CHANGELOG.md

```diff
@@ -0,0 +1,141 @@
+# Change Log
```

Comment:
@hxlaf Please delete the changelog.md file altogether, there should be no changelog since this package hasn't been in any previous releases yet.

Reply:
> @hxlaf Please delete the changelog.md file altogether, there should be no changelog since this package hasn't been in any previous releases yet.

Thank you! Done.

Reply:
@hxlaf File appears to be still kicking ;-)